CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
-- No unreleased changes
+- Update default web_ui_endpoint handling in tasks to account for /cd4pe prefix in url.
```

Comment:
we should have headings like `### Fixed` here, too, but i don't think that's worth blocking on, especially because we're going to turn around and release this